### PR TITLE
Make changes to SEP to use two escrow accounts

### PR DESCRIPTION
**Pull request by leighmcculloch**
### What
Make changes to the SEP so that it uses two escrow accounts and no claimable balances.

### Why
See stellar/stellar-protocol#938 for why.

Intended for merging into stellar#3.

### CODE REVIEW

File: specifications/sep-payment-channel-mechanism.md [hunk 1]
```diff
@@ -108,17 +106,18 @@ The participants may agree to change the period O at anytime by following the
 The payment channel utilizes two Stellar accounts that are both 2-of-2
```
Comment (acharb) [inline specifications/sep-payment-channel-mechanism.md:106]: 💡  Thoughts on changing this section header to `Escrow Accounts`? To differentiate from regular accounts I and R

File: specifications/sep-payment-channel-mechanism.md [hunk 2]
```diff
@@ -235,33 +224,31 @@ To update the payment channel state, the participants:
...
-  account V as a sponsor of future reserves.
-  - One `CREATE_CLAIMABLE_BALANCE` operation for each trustline that is
-  disbursing funds to R.
-  - One `END_SPONSORING_FUTURE_RESERVES` operation that confirms reserve
-  account V's sponsorship.
-  - One or more `SET_OPTIONS` operation adjusting escrow account E's
-  thresholds to give I full control of E, and removing R's signers.
-  - One or more `SET_OPTIONS` operation adjusting reserve account E's
-  thresholds to give R full control of V, and removing I's signers.
+  - One `PAYMENT` operation for each trustline that is disbursing funds to EI to
```
Comment (acharb) [inline specifications/sep-payment-channel-mechanism.md:240]:
```suggestion
  - One `PAYMENT` operation for each trustline that is disbursing funds from EI to
```

**Review by acharb — APPROVED** [review]
Looks great 💯 . 

I really like w/ this implementation C_i doesn't create any new ledger entries, so the risk of not having enough funds to sponsor is gone. Because of that we can remove [this paragraph](https://github.com/leighmcculloch/stellar--experimental-payment-channels/pull/1/files#diff-b01917f0debe4bb9e19259c6e34888dd6c41d4f2da28f49af7fc820ba5a38dc7R561) right? Since that security concern is now gone

Going to approve since non-blocking, and from my understanding this 2-escrow account idea looks good

File: specifications/sep-payment-channel-mechanism.md (hunk 2 above)
Comment (leighmcculloch) [reply specifications/sep-payment-channel-mechanism.md]: Thanks! de883a6

File: specifications/sep-payment-channel-mechanism.md (hunk 1 above)
Comment (leighmcculloch) [reply specifications/sep-payment-channel-mechanism.md:106]: Good idea! 53613ec

**Comment by leighmcculloch** [comment]
This PR has been migrated to https://github.com/stellar/experimental-payment-channels/pull/13.

**Pull request closed without merging**